extensions/wikia/Forum/ForumController.class.php

```diff
@@ -65,8 +65,6 @@ public function board() {
 
 		// TODO: keep the varnish cache and do purging on post
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)